src/core/proj/qgscoordinatetransform.h

```diff
@@ -142,6 +150,20 @@ class CORE_EXPORT QgsCoordinateTransform
 
     ~QgsCoordinateTransform();
 
+    /**
+     * Returns TRUE if it is theoretically possible to transform between \a source and \a destination CRSes.
+     *
+     * For example, will return FALSE if \a source and \a destination relate to different celestial bodies and
+     * a transformation between them will never be possible.
+     *
+     * \warning This method tests only if it is theoretically possible to transform between the CRSes, not whether a
+     * transform can actually be constructed on the system. It is possible that this method may return TRUE,
+     * yet construction of a matching QgsCoordinateTransform fails (e.g. due to missing grid shift files on the system).
+     *
+     * \since QGIS 3.26
+     */
+    static bool transformationIsPossible( const QgsCoordinateReferenceSystem &source, const QgsCoordinateReferenceSystem &destination );
```

Comment:
Wouldn't the idiom `isTransformationPossible()` be more in-line with existing bool functions in the codebase?

Reply:
Good point, changed